### PR TITLE
adds dotenv for virtual environment variable configurations in ga4 lambda

### PR DESCRIPTION
**Pull request by ghepting**
## Purpose
provide a way to specify required environment variable in virtual environments (i.e. development)

## Approach
uses dotenv npm package to load environment variables into express.js from local gitignored .env file

## Dependencies and/or References
- https://www.npmjs.com/package/dotenv

## Deployment
- [ ] ensure environment variables are set in circleCI and properly configured in production lambda

### CODE REVIEW

File: apps/google-analytics-4/lambda/config/secrets.dev.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+signingSecret: mFjgMwlcE5iHKl855rGk9adWTP9ZagSamQctAElpptMtpGQmNIuGoRWOZZ_lhU_4
```
Comment (ghepting) [inline apps/google-analytics-4/lambda/config/secrets.dev.yml:1]: bah, renamed this and exposes this secret. issuing a new signing secret now

Comment (ghepting) [reply apps/google-analytics-4/lambda/config/secrets.dev.yml:1]: done

Comment (ghepting) [reply apps/google-analytics-4/lambda/config/secrets.dev.yml]: new secret is updated in 1password @contentful/team-integrations

File: apps/google-analytics-4/lambda/config/secrets.test.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+signingSecret: ${env:SIGNING_SECRET}
```
Comment (ryunsong-contentful) [inline apps/google-analytics-4/lambda/config/secrets.test.yml:1]: Does test === staging?

**Review by ryunsong-contentful — APPROVED** [review]

**Pull request closed without merging**